### PR TITLE
fix:keyboard doesn't slide back down after clicking on the save option

### LINKED ISSUE #299: The keyboard doesn't slide back down after clicking on the save option
**Summary:**

The keyboard doesn't slide back down after clicking on the save option after editing in the text field in Edit Profile fragament

**Steps to reproduce:**

Go to Profile Activity
Click on Edit text in any field, edit something
Click on save button

**Expected behavior:**

Keyboard should slide down

**Observed behavior:**

The keyboard is not sliding down

**Device and Android version:**

Realme 2 Pro
Android version 8.1.0

**Screenshots:**

videotogif_2019 02 13_16 22 28 1

### PR DESCRIPTION
**Pull request by shaliniv16**
Fixes #299 

**Screenshots**

videotogif_2019 02 14_20 49 27 1

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

### CODE REVIEW

**Comment by shaliniv16** [comment]
@ankurs287 @luckyman20 please review.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 1]
```diff
@@ -261,10 +261,15 @@ public void hideKeyboard() {
...
-            view = new View(this);
+        if (imm != null) {
+
+            if (view == null) {
+                view = new View(this);
+            }
+            
             imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
         }
+
```
Comment (codepoet2017390) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java]: Please remove this extra space

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 2]
```diff
@@ -261,10 +261,15 @@ public void hideKeyboard() {
...
                 = (InputMethodManager) getSystemService(Activity.INPUT_METHOD_SERVICE);
         View view = getCurrentFocus();
-        if (view == null && imm != null) {
-            view = new View(this);
+        if (imm != null) {
+
+            if (view == null) {
+                view = new View(this);
+            }
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java]:
```suggestion
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 3]
```diff
@@ -261,10 +261,15 @@ public void hideKeyboard() {
         InputMethodManager imm
                 = (InputMethodManager) getSystemService(Activity.INPUT_METHOD_SERVICE);
         View view = getCurrentFocus();
-        if (view == null && imm != null) {
-            view = new View(this);
+        if (imm != null) {
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java]:
```suggestion
```

**Comment by luckyman20** [comment]
@shaliniv16 Since you haven't resolved the comments yet. Closing this PR.

**Pull request closed without merging**